sdk/textanalytics/azure-ai-textanalytics/src/test/java/com/azure/ai/textanalytics/TextAnalyticsClientTestBase.java

```
@@ -0,0 +1,428 @@
+     * @param testApi the API to test.
+     */
+    private <T> void validateDocuments(DocumentResultCollection<T> expectedResult,
+                                       DocumentResultCollection<T> actualResult, String testApi) {
+        switch (testApi) {
+            case "Language":
+                final List<DetectLanguageResult> expectedResultList = expectedResult.stream()
+                    .filter(element -> element instanceof DetectLanguageResult)
+                    .map(element -> (DetectLanguageResult) element)
+                    .collect(Collectors.toList());
+
+                final List<DetectLanguageResult> actualResultList = actualResult.stream()
+                    .filter(element -> element instanceof DetectLanguageResult)
+                    .map(element -> (DetectLanguageResult) element)
+                    .collect(Collectors.toList());
+                assertEquals(expectedResultList.size(), actualResultList.size());
+
+                actualResultList.forEach(actualItem -> {
+                    Optional<DetectLanguageResult> optionalExpectedItem = expectedResultList.stream().filter(
+                        expectedEachItem -> actualItem.getId().equals(expectedEachItem.getId())).findFirst();
+                    assertTrue(optionalExpectedItem.isPresent());
+                    DetectLanguageResult expectedItem = optionalExpectedItem.get();
+                    if (actualItem.getError() == null && this.showStatistics) {
+                        validatePrimaryLanguage(expectedItem.getPrimaryLanguage(), actualItem.getPrimaryLanguage());
+                        validateDocumentStatistics(expectedItem.getStatistics(), actualItem.getStatistics());
+                        validateDetectedLanguages(expectedItem.getDetectedLanguages(), actualItem.getDetectedLanguages());
+                    }
+                });
+                break;
+            default:
+                break;
```

Default, you should throw an exception rather than passing successfully. If I put testAPI, "language", the test case is written wrong and i should throw.